### PR TITLE
chore(ci): add docker build test

### PR DESCRIPTION
**Pull request by ShaunSHamilton**
I have no idea what I am doing

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Testing a test

### CODE REVIEW

**Comment by ShaunSHamilton** [comment]
I am happy with this. I am also happy just merging this myself as this is low-impact.

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    name: Dock
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout Source Files
+        uses: actions/checkout@v3
+
+      - name: Build Dockerfile
+        run: |
+          docker build -f Dockerfile -t euler-rust .
```
Comment (ojeytonwilliams) [inline .github/workflows/build-test.yml:18]:
```suggestion
        run: docker build -t euler-rust .
```
If the Dockerfile is called Dockerfile Docker will find the file.

Comment (ShaunSHamilton) [reply .github/workflows/build-test.yml:18]: I know, but I like being explicit with cmd-line stuffs. May I keep it?

Comment (ojeytonwilliams) [reply .github/workflows/build-test.yml:18]: Sure, it's pretty much preference.  I slightly prefer brevity, but this fine.

**Review by naomi-lgbt — APPROVED** [review]
Action looks good to me.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM, too

**Pull request merged**